main.py

```diff
@@ -0,0 +1,10 @@
+import requests
```

Comment:
lab 5 implementation is expected to be in `lab_5_scrapper/scrapper.py` file

Reply:
Done